meetings/160330_organisers.md

```diff
@@ -1,11 +1,172 @@
+
+Halfdan said we would also send the Pre-workshop questionnaire in advance. For the last Software Carpentry workshop they got information the day before; a bit too late to adjust the lessons.
+
+We discussed whether we should have the pre-workshop questionnaire with the registration and Karin clearly stated it was a bad idea as it would scare a lot of people (they have tried before!).
+We therefore agreed to send the pre-workshop questionnaire two weeks before.
+
+The procedure needs to be updated:
+
+
+- Admin to generate links (with the workshop) for questionnaire. 
+- The Library generates nettskjema for the registrations and questionnaires and the lead instructors got access. 
+
+#### Actions ####
+
+- Carrie to organize the one-day R workshop for the 10th of May. Both Carrie and Christian will look at the R materials on the Software Carpentry website and they will meet to decide on the content of R workshop. 
+- Carrie is in charge of finding helpers (Anne said she can be a helper for this workshop)
+- Lex will send an email to get a lead instructor for the 20/21th June Software Carpentry workshop
+- Lex or Karin to update the UIO instructions for organizing a workshop at UIO.
+
+## Workshop "format" ##
+
+We have discussed the current Software & Data Carpentry workshop format (2 days workshops). 
+We all agreed that it is still necessary to organise 2 days workshops few times a year, especially if we open our workshops to other Universities in Norway & other Nordic countries. At the same time, there is a clear need for shorter workshops (1/2 day or 1 full day only) where we could either introduce one subject or follow-up on Software Carpentry lessons. Some suggestions were given:
+
+- one day on git where we discuss collaborative work (branching, conflict resolution, etc.)
+- 1/2 day on Shell where we could teach the entire Software Carpentry Lesson
+- 1/2 day on licensing (software and data license)
+- 1/2 or 1 day introducing Python or R (could be useful for Data Carpentry follow-up workshops)
+- 1/2 or 1 day Software Carpentry workshop for professors
+
+Vincent also mentionned the interest of ForBio (http://www.forbio.uio.no) to develop a Software Carpentry workshop week. The ambition of Forbio is to train a new generation of biosystematists in the Nordic countries and as many PhDs are coming from abroad, it would be necessary to hold a full week rather than several 1/2 days or 1 day workshops. 
+
+We discussed the possibility to participate more actively in Research school to train younger researchers (PhDs), master Students or even undergraduates. We all agreed there is a need to adapt Software Carpentry workshop "format".
+
+The codata international Training Workshop format was discussed: they offer the possibility to build your own workshop with lessons chosen according to your needs. We all think we could use the same kind of format, with adequate information on the possible prerequisites for each session.

```

Comment:
You could link to and example workshop: http://www.codata.org/news/105/62/CODATA-International-Training-Workshop-in-Big-Data-for-Science-Beijing-4-17-July